raspbian/applications/rpi-update.md

```diff
@@ -0,0 +1,37 @@
+# rpi-update
+
+`rpi-update` is a command line application that will update your Raspbian kernel and Videocore firmware to the latest pre-release versions.
+
+**WARNING: Pre-release versions of software are not guaranteed to work. You should not use `rpi-update` on any system unless recommended to do so by a Raspberry Pi engineer. It may leave your system unreliable or even completely broken. It should not be used as part of any regular update process**
```

Comment:
Is it worth advising creating a backup of the SD card first? (Or is that implicit in the "It may leave your system ... completely broken" warning?)

Reply:
Have added a line in the "running" section